barretenberg/cpp/src/barretenberg/stdlib/primitives/plookup/plookup.test.cpp

```diff
@@ -191,7 +192,7 @@ TEST(stdlib_plookup, uint32_xor)
 
     const auto lookup = plookup_read::get_lookup_accumulators(MultiTableId::UINT32_XOR, left, right, true);
 
-    const auto left_slices = numeric::slice_input(left_value, 1 << 6, num_lookups);
+    const auto left_slices = numeric::slice_input(left_value, 2 << 6, num_lookups);
```

Comment:
I suppose it's this change that broke `stdlib_plookup.uint32_xor`

Reply:
ugh woops i took a note I did this then never fixed it. Glad we have tests that catch this :  D